src/components/LicenseRemindModal/index.jsx

```diff
@@ -0,0 +1,249 @@
+            />
+            <Field
+              id="email-template-body"
+              name="email-template-body"
+              component={TextAreaAutoSize}
+              label="Body"
+              disabled
+            />
+            <Field
+              id="email-template-closing"
+              name="email-template-closing"
+              component={TextAreaAutoSize}
+              label="Customize Closing"
+            />
+          </div>
+        </form>
+      </React.Fragment>
+    );
+  }
+
+  renderErrorMessage() {
+    const { error } = this.props;
+
+    return (
+      <div
+        ref={this.errorMessageRef}
+        tabIndex="-1"
+      >
+        <StatusAlert
+          alertType="danger"
+          iconClassName="fa fa-times-circle"
+          title="Unable to send reminder email"
+          message={error.length > 1 ? (
+            <ul className="m-0 pl-4">
+              {error.map(message => <li key={message}>{message}</li>)}
+            </ul>
+                    ) : (
+                        error[0]
+                    )}
+        />
+      </div>
+    );
+  }
+  renderTitle() {
+    const { title } = this.props;
+    return (
+      <React.Fragment>
+        <span className="d-block">{title}</span>
+        <small>
+          subtitle
+        </small>
+      </React.Fragment>
+    );
+  }
+  render() {
+    const {
+      onClose,
+      submitting,
+      handleSubmit,
+    } = this.props;
+    return (
+      <React.Fragment>
+        <Modal
+          ref={this.modalRef}
+          title={this.renderTitle()}
+          body={this.renderBody()}
+          buttons={[
+            <Button
+              key="license-remind-submit-btn"
+              disabled={submitting}
+              className="license-remind-save-btn btn-primary"
+              onClick={handleSubmit(this.handleModalSubmit)}
+            >
+              <React.Fragment>
+                {submitting && <Icon className="fa fa-spinner fa-spin mr-2" />}
+                {'Send Reminder'}
+              </React.Fragment>
+            </Button>,
+          ]}
+          closeText="Cancel"
+          onClose={onClose}
+          open
+        />
+      </React.Fragment>
+    );
+  }
+}
+LicenseRemindModal.defaultProps = {
+  error: null,
+  isBulkRemind: false,
+  user: {},
+  pendingUsersCount: 0,
+};
+LicenseRemindModal.propTypes = {
+  // props From redux-form
+  handleSubmit: PropTypes.func.isRequired,
+  submitting: PropTypes.bool.isRequired,
+  submitSucceeded: PropTypes.bool.isRequired,
+  submitFailed: PropTypes.bool.isRequired,
+  error: PropTypes.arrayOf(PropTypes.string),
+  initialize: PropTypes.func.isRequired,
+  // custom props
+  title: PropTypes.string.isRequired,
+  onClose: PropTypes.func.isRequired,
+  onSuccess: PropTypes.func.isRequired,
+  sendLicenseReminder: PropTypes.func.isRequired,
+  isBulkRemind: PropTypes.bool,
+  pendingUsersCount: PropTypes.number,
+  user: PropTypes.shape({
```

Comment:
wouldn't it be better to receive `userEmail` as a prop instead of the object?

Reply:
The whole object has been passed as when we need to call the api then might be there is need of UUID as well.